src/sidebar/store/modules/test/selection-test.js

```diff
@@ -1,15 +1,16 @@
 'use strict';
 
+const annotations = require('../annotations');
 const createStore = require('../../create-store');
 const selection = require('../selection');
 const uiConstants = require('../../../ui-constants');
 
-describe('selection', () => {
+describe('Selection Store', () => {
```

Comment:
We have a number of different naming schemes for the top-level describe block in tests. This was discussed in Slack a while back before you joined, but I would like us to standardize on just a few:

- If the module exports one function/class, then the name of that function/class is fine providing it is reasonably likely to be unique (eg. `MyReactComponent`)
- If the module has a basename that is reasonably likely to be unique, then that's OK (eg. `build-thread`)
- Otherwise, use the path as if you were importing it from `src` eg. 'store/modules/selection'.

Having more consistent naming should make it easier to know what files to look at when looking at a list of test failures.